extensions/wikia/ImageReview/ImageReviewHelper.class.php

```diff
@@ -374,11 +393,39 @@ private function imageListAdditionalAction( $sType, DatabaseBase $oDB, Array $aV
 		] );
 	}
 
+	public function getImageStats() {
```

Comment:
This method return data only from cache. Is this name is obvious in file context?